server/v2/cometbft/utils.go

```
@@ -1,15 +1,21 @@
 	"errors"
 	"fmt"
 	"math"
+	"slices"
 	"strings"
 	"time"
 
 	abci "github.com/cometbft/cometbft/api/cometbft/abci/v1"
 	cmtproto "github.com/cometbft/cometbft/api/cometbft/types/v1"
+	cryptoenc "github.com/cometbft/cometbft/crypto/encoding"
+	protoio "github.com/cosmos/gogoproto/io"
+	"github.com/cosmos/gogoproto/proto"
```

golangci-lint (1.62.2)

ST1019: package "github.com/cosmos/gogoproto/proto" is being imported more than once

(stylecheck)